src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -389,6 +391,11 @@ def file_issue(testcase,
                                       security_severity):
       issue.labels.add(result)
 
+  # Add additional body text from metadata.
+  additional_fields = json.loads(testcase.get_metadata('additional_fields', ''))
+  for key, value in six.iteritems(additional_fields):
```

Comment:
not need to use six here. just do `additional_fields.items()`. We no longer support python 2.